src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -321,9 +321,10 @@ private void LogScriptIfGiven(string type, string scriptPath)
                 .Where(s => s.BuildProperties != null)
                 .SelectMany(s => s.BuildProperties)
                 .ToDictionary(p => p.Key, p => p.Value);
+            buildProperties[ManifestFilePropertyKeys.OperationId] = context.OperationId;
```

Comment:
the `BuildScriptGeneratorContext` already has a member `IDictionary<string, string> Properties`;
so why do we need the new `BuildScriptGeneratorContext.OperationId`?

Reply:
The thing is those Properties are user supplied i.e coming from `-p` supplied at command line. We currently do not just use those properties directly as properties that go into manifest file (i think that makes sense as we might want to write or do something differently). I think the names are getting us confused. I would rather change this line of code to say `manifestFileProperties` instead.